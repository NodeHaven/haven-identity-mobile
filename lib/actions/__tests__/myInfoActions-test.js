// Copyright (C) 2018 ConsenSys AG
//
// This file is part of uPort Mobile App.
//
// uPort Mobile App is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
//
// uPort Mobile App is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
//
// You should have received a copy of the GNU General Public License
// along with uPort Mobile App.  If not, see <http://www.gnu.org/licenses/>.
//

import {editMyInfo, updateShareToken, saveShareToken} from '../myInfoActions.js'

it('creates a EDIT_MY_INFO action', () => {
  expect(editMyInfo({name: 'Bob Bixby'})).toMatchSnapshot()
})

it('creates a UPDATE_SHARE_TOKEN action', () => {
  expect(updateShareToken('0x01234')).toMatchSnapshot()
})

it('creates a SAVE_SHARE_TOKEN action', () => {
  expect(saveShareToken('0x01234', 'JWT')).toMatchSnapshot()
})
