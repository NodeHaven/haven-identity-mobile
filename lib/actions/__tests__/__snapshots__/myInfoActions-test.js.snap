// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a EDIT_MY_INFO action 1`] = `
Object {
  "change": Object {
    "name": "Bob Bixby",
  },
  "type": "EDIT_MY_INFO",
}
`;

exports[`creates a SAVE_SHARE_TOKEN action 1`] = `
Object {
  "address": "0x01234",
  "token": "JWT",
  "type": "SAVE_SHARE_TOKEN",
}
`;

exports[`creates a UPDATE_SHARE_TOKEN action 1`] = `
Object {
  "address": "0x01234",
  "type": "UPDATE_SHARE_TOKEN",
}
`;
